lsp-mode.el

```diff
@@ -3354,8 +3357,10 @@ in that particular folder."
                                   (error nil)))
    (when (featurep 'project)
      (when-let ((project (project-current)))
-       (car (with-no-warnings
-              (project-roots project)))))))
+       (if (fboundp 'project-root)
+           (car (project-root project))
```

Comment:
on my system `(project-root (project-current))` returns a string?

Reply:
Yes, it looks like it. Better remove that "car" then?

Reply:
Yep.

Reply:
removed the car